Sources/Utils/OptParse.cpp

```diff
@@ -267,11 +267,18 @@ void OptParse::usageDie(char const *format, ...) {
   help_align += 2;
 
   for (auto const &e : _options) {
-    if (e.second.hidden)
+    if (e.second.hidden) {
       continue;
+    }
 
-    Print("  -%c, --%s", e.second.shortName, e.first.c_str());
-    Print(" %s", (e.second.type == stringOption ? "ARG" : "   "));
+    static std::map<OptionType, char const *> const argTypePlaceholder = {
+        {stringOption, "ARG        "},
+        {vectorOption, "ARG        "},
```

Comment:
Given that you're making these const-length strings (ie space-padded), it's probably easier to understand if you align them.